src/main.js

```diff
@@ -29,15 +28,14 @@ const createWindow = () => {
   ipcMain.on("begin-session", (...args) => {
     win.setMinimumSize(800, 600);
     win.setSize(800, 600);
-    beginSession(...args);
+    sessionId = beginSession(...args);
```

Comment:
What is the sessionId used for here? I don't see any other place that you added it here